busco/5.4.7/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM ubuntu:focal as app
+
+ARG BUSCO_VER="5.4.7"
+ARG BBMAP_VER="39.01"
+ARG BLAST_VER="2.14.0"
+ARG DEBIAN_FRONTEND=noninteractive
+
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="BUSCO"
+LABEL software.version="${BUSCO_VER}"
+LABEL description="Assessing genome assembly and annotation completeness with Benchmarking Universal Single-Copy Orthologs"
+LABEL website="https://busco.ezlab.org/"
+LABEL license="https://gitlab.com/ezlab/busco/-/raw/master/LICENSE"
+LABEL maintainer="Kutluhan Incekara"
+LABEL maintainer.email="kutluhan.incekara@ct.gov"
+
+# install dependencies
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    wget \
+    python3-biopython \
+    python3-pandas \
+    python3-setuptools\ 
+    hmmer \   
+    prodigal \    
+    augustus \
+    r-cran-ggplot2 \      
+    gcc-x86-64-linux-gnu \
+    default-jre \ 
+    libjenkins-json-java \
+    libgoogle-gson-java \
+    libjson-java \
+    && rm -rf /var/lib/apt/lists/* && apt-get autoclean \
+    && ln -s /usr/bin/python3 /usr/bin/python
+
+# install blast, bbtools, sepp, metaeuk, and busco
+RUN wget https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.14.0/ncbi-blast-${BLAST_VER}+-x64-linux.tar.gz &&\
+    tar -xvf ncbi-blast-${BLAST_VER}+-x64-linux.tar.gz && rm ncbi-blast-${BLAST_VER}+-x64-linux.tar.gz &&\
+    wget https://raw.github.com/smirarab/sepp-refs/master/gg/sepp-package.tar.bz &&\
+    tar xvfj sepp-package.tar.bz && rm sepp-package.tar.bz &&\
+    cd sepp-package/sepp &&\
+    python setup.py config -c && chmod 755 run_* && cd / &&\
+    wget https://sourceforge.net/projects/bbmap/files/BBMap_${BBMAP_VER}.tar.gz &&\
+    tar -xvf BBMap_${BBMAP_VER}.tar.gz && rm BBMap_${BBMAP_VER}.tar.gz &&\    
+    mv /bbmap/* /usr/local/bin/ &&\
+    wget https://mmseqs.com/metaeuk/metaeuk-linux-sse41.tar.gz &&\
+    tar -xvf metaeuk-linux-sse41.tar.gz && rm metaeuk-linux-sse41.tar.gz &&\
+    mv  /metaeuk/bin/* /usr/local/bin/ &&\
+    wget https://gitlab.com/ezlab/busco/-/archive/${BUSCO_VER}/busco-${BUSCO_VER}.tar.gz &&\
+    tar -xvf busco-${BUSCO_VER}.tar.gz && \
+    rm busco-${BUSCO_VER}.tar.gz &&\
+    cd busco-${BUSCO_VER} && \
+    python3 setup.py install
```

Comment:
Could you separate this layer into one per tool? (i.e. one RUN statement for blast, one for bbtools, one for sepp, one for metaeuk, and one for busco)

It makes it easier to read.

Additional things that I suggest 
- use a version with sepp (latest release was [4.5.1](https://github.com/smirarab/sepp/releases/tag/4.5.1), but there have been 8 commits since then - so perhaps you'd like to use a commit instead?)
- metaeuk likely can be downloaded with a ]version here : https://github.com/soedinglab/metaeuk. Does it still work?

Reply:
I didn't separate tools into different layers to avoid increasing size in the first place. I believe there is no meaning in this image, so I separated them.

SEPP is evil, cursed, and the mother of all problems. Only Greengenes package worked here. 
https://github.com/smirarab/sepp/blob/master/sepp-package/README.md

Changed metaeuk to release 6-a5d39d9.

Reply:
The permalink of https://raw.github.com/smirarab/sepp-refs/master/gg/sepp-package.tar.bz is https://github.com/smirarab/sepp-refs/blob/54415e8905c5fa26cdd631c526b21f2bcdba95b5/gg/sepp-package.tar.bz , which is probably more stable. Could you try that link?